VideoLocker/src/main/java/org/edx/mobile/view/adapters/CourseDiscussionResponsesAdapter.java

```diff
@@ -219,10 +219,11 @@ private void bindViewHolderToResponseRow(DiscussionResponseViewHolder holder, fi
             @Override
             public void onClick(View v) {
                 if (comment.getChildren().isEmpty()) {
-                    router.showCourseDiscussionAddComment(context, comment);
+                    if (!discussionThread.isClosed())
+                        router.showCourseDiscussionAddComment(context, comment);
```

Comment:
>Please add braces around conditional statements, or put it all in one line, according to AOSP guidelines.

Done

>Also, this condition could be evaluated outside of the click listener to just disable the add comment button.

Its in the right place, cuz, we first need to check for children before we can decide to disable/enable the button.